test/conformance/filesystem_perm_test.go

```diff
@@ -43,10 +43,10 @@ func verifyPermString(resp string, expected string) error {
 	return nil
 }
 
-func TestMustFileSystemPermissions(t *testing.T) {
-	logger := logging.GetContextLogger("TestFileSystemPermissions")
+func testFileSystemPermissions(testName string, paths map[string]FilePathInfo, t *testing.T) {
+	logger := logging.GetContextLogger(testName)
```

Comment:
Can I recommend returning error and calling `t.Error()` in the test itself?

Reply:
Done.  And let TestShouldHaveFileSystemPermissions() use t.Error() and TestMustHaveFileSystemPermissions() use t.Fatal().